app.js

```diff
@@ -11,6 +11,7 @@ const omit = require('lodash').omit;
 
 // Electron Libs
 const { app, BrowserWindow, ipcMain } = require('electron');
+const electron = require('electron');
```

Comment:
I think you can just add `screen` to the above line like so 

```js
const { app, BrowserWindow, ipcMain, screen } = require('electron');
```

Reply:
From the [API Documentation for screen](https://github.com/electron/electron/blob/master/docs/api/screen.md) you posted in the issue.

*Note: In the renderer / DevTools, window.screen is a reserved DOM property, so writing let {screen} = require('electron') will not work.*

I could try and see what happens :stuck_out_tongue: ??

Reply:
Ahh, nice catch! 👍